cmd/ui/src/views/Explore/ExploreSearch/edgeTypes.tsx

```diff
@@ -89,13 +89,17 @@ export const AllEdgeTypes: Category[] = [
             {
                 name: 'Active Directory Certificate Services',
                 edgeTypes: [
-                    ActiveDirectoryRelationshipKind.RootCAFor,
-                    ActiveDirectoryRelationshipKind.PublishedTo,
-                    ActiveDirectoryRelationshipKind.ManageCertificates,
-                    ActiveDirectoryRelationshipKind.ManageCA,
                     ActiveDirectoryRelationshipKind.DelegatedEnrollmentAgent,
                     ActiveDirectoryRelationshipKind.Enroll,
```

Comment:
I believe we only list transitive edges in this filter so the enroll edge, for instance, would not be something that should be populated here. You'll notice getChanges and getChangesAll are absent from any of the AD filters.

Reply:
Oh yeah, you are right.. Good catch. It is actually only GoldenCert, and the ESCX edges that are transitive. So maybe we should remove all the others. I will ask Andy if he agrees

Reply:
Yea that sounds correct to only include GoldenCert and the ESCX edges. Thanks for confirming with Andy